packages/frontend/src/hooks/pinning/useSpaceMutation.ts

```diff
@@ -23,6 +23,7 @@ export const useSpacePinningMutation = (projectUuid: string) => {
                     'spaces',
                     space.projectUuid,
                 ]);
+                await queryClient.invalidateQueries(['projects']);
```

Comment:
Refreshing the spaces query is faster and a more localized refresh. 
```suggestion
                await queryClient.invalidateQueries(['projects', projectUuid, 'spaces']);
```

Reply:
e2e test failed because of file format